card/lib/card/format.rb

```diff
@@ -204,7 +204,8 @@ def render view, args={}
         args = default_render_args view, args
         with_inclusion_mode view do
           Card::ViewCache.fetch(self, view, args) do
-            send "_view_#{ view }", args
+            method = method "_view_#{ view }"
+            method.arity == 0 ? method.call : method.call(args)
```

Comment:
possible that it's wiser to handle this at view definition time than here.  but it works :)